applications/body_pose_estimation/body_pose_estimation.py

```diff
@@ -347,6 +347,12 @@ def __init__(self, data, source="v4l2"):
     def compose(self):
         pool = UnboundedAllocator(self, name="pool")
 
+        # Determine if the DDS Publisher is enabled.
+        dds_common_args = self.kwargs("dds_common")
+        dds_publisher_args = dds_common_args | self.kwargs("dds_publisher")
+        enable_dds_publisher = dds_publisher_args["enable"]
+        del dds_publisher_args["enable"]
```

Comment:
The `dds_publisher_args` are used to create the operator later, but since the operator itself doesn't have an `enable` parameter then not deleting it would lead to an error when the operator constructor is called. I think the alternatives would be to (1) use a command-line parameter, or (2) use a separate `dds_publisher_controls` or something section in the yaml that would not otherwise be used for operator construction, but I personally preferred the yaml structure when using this current method.

I had also wondered if we'd want change Holoscan so that unused parameters could be ignored during operator construction instead, but I think not allowing unused parameters is probably safer to avoid bugs due to misspelled parameters.

Reply:
Thanks for the explanation -- no issue leaving as-is.